libs/payments/cart/src/index.ts

```diff
@@ -4,3 +4,4 @@
 export * from './lib/cart.types';
 export * from './lib/cart.factories';
 export * from './lib/cart.manager';
+export * from './lib/cart.service';
```

Comment:
Looks like we missed this from a prior export. Additional thought here is that perhaps we have services export their own NestJS Module so the needed Service is the only thing exported, rather than the app having to assemble it all.